src/Microsoft.DotNet.ApiSymbolExtensions/Filtering/AccessibilitySymbolFilter.cs

```diff
@@ -14,6 +14,9 @@ public class AccessibilitySymbolFilter : ISymbolFilter
         private readonly bool _includeEffectivelyPrivateSymbols;
         private readonly bool _includeExplicitInterfaceImplementationSymbols;
 
+        public bool IncludeInternalSymbols { get => _includeInternalSymbols; }
```

Comment:
```suggestion
        public bool IncludeInternalSymbols { get; }
```

and remove the underlying backing field as the compiler generates one automatically. Same for `IncludeEffectivelyPrivateSymbols`